task_1_Homework6.py

```diff
@@ -0,0 +1,66 @@
+﻿""" Создайте  модель из жизни.
```

Comment:
insufficient interaction in models

Reply:
Please tell me what needs to be changed or added?